pkg/project/expander_test.go

```diff
@@ -130,14 +123,10 @@ func TestExpandTopLevel(t *testing.T) {
 func TestExpandProjectPlatformOs(t *testing.T) {
```

Comment:
The update to this test seems to now be asserting something that is different from the intend. I think we can just drop this test? It was specifically testing the platform property, which has been removed, so it no longer serves a purpose.

Reply:
Good catch.